compiler/passes.nim

```diff
@@ -37,6 +36,11 @@ proc skipCodegen*(config: ConfigRef; n: PNode): bool {.inline.} =
   # something with `n`. Currently, this ignores `n` and uses the global
   # error count instead.
   result = config.errorCounter > 0
+  # no point in even attempting codegen on a nil pnode
+  result = result or n == nil
+  when not defined(release):
```

Comment:
just `doAssert n!=nil` or `assert n!=nil` depending on whether this is a hotpsot (i doubt it is)

Reply:
Obviously temporary.